src/fides/api/service/connectors/sql_connector.py

```diff
@@ -115,6 +124,18 @@ def default_cursor_result_to_rows(results: LegacyCursorResult) -> List[Row]:
             rows.append({col[0]: row_tuple[count] for count, col in enumerate(columns)})
         return rows
 
+    @staticmethod
+    def get_namespace_meta(db: Session, dataset: str) -> Optional[Dict[str, Any]]:
+        """
+        Util function to return the namespace meta for a given ctl_dataset.
+        """
+
+        return db.scalar(
+            select(CtlDataset.fides_meta["namespace"].cast(JSONB)).where(
```

Comment:
Just queries for the `fides_meta` column so we don't return a huge payload from the database for large datasets